manifests/ssltools/nssdb.pp

```diff
@@ -34,17 +24,10 @@
     group  => $group,
     mode   => '0640',
   } ->
-  exec { 'create-nss-db':
-    command => "certutil -N -d '${nss_db_dir}' -f '${nss_db_password_file}'",
-    path    => '/usr/bin',
-    umask   => '0027',
-    group   => $group,
-    creates => $nssdb_files,
-    require => Package['nss-tools'],
-  } ->
-  file { $nssdb_files:
-    owner => 'root',
-    group => $group,
-    mode  => '0640',
+  nssdb { $nss_db_dir:
+    password_file => $nss_db_password_file,
```

Comment:
You'll want an `ensure => present` for the first line of the resource since its an `ensurable` type.